extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -545,7 +545,7 @@ class SegmentationService extends PubSubService {
       volumeId: segmentationId,
       targetBuffer: {
         type: 'Uint8Array',
-        sharedArrayBuffer: window.SharedArrayBuffer,
+        sharedArrayBuffer: window.config?.useSharedArrayBuffer,
```

Comment:
I think config.useSharedArrayBuffer is a string, that can be 'FALSE', 'AUTO' , 'TRUE' so it is not a boolean

Reply:
Ah I see my bad, I have updated to handle as a string, having a getter in Cornerstone could reduce code duplication.

To be honest, I'm not 100% sure I understand this problem. Having the old code with` window.SharedArrayBuffer` definitely causes an issue. Even running OHIF master branch, connected to your PACS with no config modifications on localhost loading a seg gave me a CORS type error and crash. But using my new method of pulling from the config it seems to work with true and false values.

My team needs a change like this so I have implemented it. If you want to look further into the issue or wait for others who rely on disabling the SharedArrayBuffer to report that's ok too.